### DISCUSSION TITLE
TiddlyDesktop for Linux on-boarding suggestion

### DISCUSSION THREAD

**Discussion by villares** [TiddlyWiki/TiddlyWiki5#7781] [Ideas]
I'm new to TW and I'm enjoying it, so I'd like to share my initial confusion after downloading *TiddlyDesktop* from some getting started page I don't quite remember which. I unzipped the package and couldn't find out how to get it running. It took me a long time, and I nearly gave up, before I could find https://github.com/TiddlyWiki/TiddlyDesktop/#download-and-install and figure I should execute `nw`.
 
Now in hindsight I know that page would be the obvious place to find install & run instructions, but I'm afraid other people like me can be clumsy and could certainly use a *README* file with instructions inside the zipped package. 

Now another naive question: Why is it called `nw` ? Is it "new wiki"? Could/should it be renamed `TiddlyDesktop` and/or have an alias?

PS: Now I see why it is called `nw`, it's because it comes from https://github.com/nwjs/nw.js ... but the question remains. Could/should it be renamed or have a visible shortcut with a more ergonomic name?

**Comment by pmario** [comment]
@Jermolene IMO a short GettingStarted info could be part of the release page too.  https://github.com/TiddlyWiki/TiddlyDesktop/releases

**Comment by linonetwo** [comment]
I don't know why nw requires Terminal to run, it is a GUI library, should create installers, and double-click to run.

See TidGi-Desktop https://github.com/tiddly-gittly/TidGi-Desktop/releases , I have created some installers and download pages https://tidgi.fun/#Download that try my best to reduce new comer confusions.
